<launch>
  <rosparam command="load" file="$(find vda5050_connector)/config/daemons_params.yaml" />
  <node name="order_daemon" pkg="vda5050_connector" type="order_daemon" output="screen" >  </node>
</launch>